.github/workflows/commit_check.yaml

```
@@ -1,67 +1,174 @@
-name: General Test and Deploy
-on: push
+name: CI Pipeline
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)